saleor/graphql/tax/mutations/tax_exemption_manage.py

```diff
@@ -0,0 +1,103 @@
+import graphene
+from django.core.exceptions import ValidationError
+
+from ....checkout.fetch import fetch_checkout_info, fetch_checkout_lines
+from ....checkout.models import Checkout as Checkout
+from ....checkout.utils import invalidate_checkout_prices
+from ....core.permissions import CheckoutPermissions
+from ....graphql.checkout.types import Checkout as CheckoutType
+from ....graphql.core.mutations import BaseMutation
+from ....graphql.order.types import Order as OrderType
+from ....order import ORDER_EDITABLE_STATUS
+from ....order.models import Order as Order
+from ....tax import error_codes
+from ...core.types import Error
+
+TaxExemptionManageErrorCode = graphene.Enum.from_enum(
+    error_codes.TaxExemptionManageErrorCode
+)
+
+
+class TaxExemptionManageError(Error):
+    code = TaxExemptionManageErrorCode(description="The error code.", required=True)
+
+
+TAXABLE_OBJECTS_MAP = {Order: OrderType, Checkout: CheckoutType}
+
+
+class TaxableObject(graphene.Union):
+    class Meta:
+        types = tuple(TAXABLE_OBJECTS_MAP.values())
+
+    @classmethod
+    def resolve_type(cls, instance, info):
+        instance_type = type(instance)
+        return TAXABLE_OBJECTS_MAP[instance_type]
+
+
+class TaxExemptionManage(BaseMutation):
+    taxable_object = graphene.Field(TaxableObject)
+
+    class Arguments:
+        id = graphene.ID(description="ID of the Checkout or Order object.")
+        tax_exemption = graphene.Boolean()
+
+    class Meta:
+        description = "Exempt taxes for Checkout or Order."
```

Comment:
Lack of `ADDED_IN_X` label and `PREVIEW_FEATURE`.

Reply:
This need to be more descriptive e.g. something like this: "Exempt checkout or order from charging the taxes. When tax exemption is enabled, taxes won't be charged for the checkout or order. Taxes may still be calculated in cases when product prices are entered with the tax included and the net price needs to be known."